hue/templates/configmap.yaml

```diff
@@ -191,15 +138,15 @@ data:
       livy_server_session_kind={{ .Values.config.spark.livyServerSessionKind }}
 
       # Whether Livy requires client to perform Kerberos authentication.
-      security_enabled={{ .Values.config.spark.securityEnabled }}
+      #security_enabled={{ .Values.config.spark.securityEnabled }}
 
       # Host of the Sql Server
-      sql_server_host={{ .Values.config.spark.sqlServerHost }}
+      #sql_server_host={{ .Values.config.spark.sqlServerHost }}
 
       # Port of the Sql Server
-      sql_server_port={{ .Values.config.spark.sqlServerPort }}
+      #sql_server_port={{ .Values.config.spark.sqlServerPort }}
```

Comment:
if you want to remove this, you should do it via a conditional

Reply:
Same thing as the previous one.